packages/language-server/test/completion-items-spec.ts

```diff
@@ -11,7 +11,8 @@ import { computeLSPKind } from "../src/completion-items";
 import {
```

Comment:
What % of time did you work on the productive logic versus the tests?
Feels like we could use snapshot tests here.
Maybe I could evaluate those with the diagnostics scenario first.